### ISSUE TITLE
Remove authorization headers when exporting collections

### ISSUE THREAD

**Issue by chrisdeso** [a85/POSTMan-Chrome-Extension#5092]
**Is your feature request related to a problem? Please describe.**
Upon exporting a collection that has base64-encoded credentials passed as a header, those credentials are also exported.

**Describe the solution you'd like**
The ability to omit these sensitive headers/credentials in the exported file without needing to remove the headers from the saved request.

**Describe alternatives you've considered**
Leveraging variables to store the headers is a way to do it but it would be great to have a way to flag/omit these before exporting≥

**Comment by dellagustin** [comment]
I actually think that omitting Authorization headers should be default behavior, that could in turn be overridden per collection / request (the more granular setting wins).

**Comment by NickBower-Woodside** [comment]
5 years later and Postman is still bundling tokens and other secrets in exported collections it seems.

**Comment by mrutyunjay-swain** [comment]
> 5 years later and Postman is still bundling tokens and other secrets in exported collections it seems.

yes they still do, had to manually delete them from the saved/exported collection in disk, before sharing those collections. Painful and risky process all the time.
